pinot-core/src/main/java/org/apache/pinot/core/transport/ServerRoutingInstance.java

```diff
@@ -19,42 +19,48 @@
 package org.apache.pinot.core.transport;
 
 import com.google.common.net.InternetDomainName;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
 import javax.annotation.concurrent.ThreadSafe;
-
-import static org.apache.pinot.common.utils.CommonConstants.Helix.PREFIX_OF_SERVER_INSTANCE;
-import static org.apache.pinot.common.utils.CommonConstants.Helix.TableType;
+import org.apache.pinot.common.utils.CommonConstants.Helix;
+import org.apache.pinot.common.utils.CommonConstants.Helix.TableType;
 
 
 /**
- * The {@code Server} class represents the routing target instance which contains the information of host name, port,
- * and table type it serves.
+ * The {@code ServerRoutingInstance} class represents the routing target instance which contains the information of
+ * hostname, port, and table type it serves.
  * <p>Different table types on same host and port are counted as different instances. Therefore, one single Pinot Server
  * might be treated as two different routing target instances based on the types of table it serves.
  */
 @ThreadSafe
-public class Server {
-  private static final String NAME_PORT_DELIMITER = "_";
+public class ServerRoutingInstance {
+  private static final String HOSTNAME_PORT_DELIMITER = "_";
```

Comment:
Nit: HOST_PORT_DELIMITER?

Reply:
Based on wikipedia, hostname is usually used as a single word, so will keep it this way :-P